Test/Tests/Component/NodeManager.Tests.ps1

```diff
@@ -148,11 +151,21 @@ Describe "Node manager" {
             "MultiNode",
             Justification="It's actually used."
         )]
-        $MultiNode = New-MultiNodeSetup -TestenvConfFile $TestenvConfFile -InstallNodeMgr
+        $MultiNode = New-MultiNodeSetup -TestenvConfFile $TestenvConfFile
+
+        foreach ($Session in $MultiNode.Sessions) {
+            Install-Components -Session $Session
```

Comment:
Is it possible to use our brand-new shiny {Initialize,Clear}-ComputeNode in this test?

Reply:
Yes. But if we want to invoke merge-logs witout DontCleanup flag then services have to be stopped after every test and started before every test, so it will be redundant to start then in BeforeAll clause.

Reply:
Something is wrong with the logic here. We do want to have logging per test case, even if services are not restarted between test cases. 

Regarding initialization Initialize-ComputeNode, it seems like it's only useful for smoke tests. Since nodemgr tests aren't smoke, maybe we shouldn't try to force Initialize-ComputeNode usage into them.

Reply:
http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/2737/TestReports/WindowsCompute/detailed_logs/Tunneling%20with%20Agent%20tests.Tunneling%20MPLSoGRE.ICMP%20-%20Ping%20between%20containers%20on%20separate%20compute%20nodes%20succeeds.txt

http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/2737/TestReports/WindowsCompute/detailed_logs/Tunneling%20with%20Agent%20tests.Tunneling%20MPLSoGRE.IP%20fragmentation%20-%20ICMP%20-%20Ping%20with%20big%20buffer%20succeeds.txt

http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/2737/TestReports/WindowsCompute/detailed_logs/Tunneling%20with%20Agent%20tests.Tunneling%20MPLSoGRE.IP%20fragmentation%20-%20UDP%20-%20sending%20big%20buffer%20succeeds.txt

http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/2737/TestReports/WindowsCompute/detailed_logs/Tunneling%20with%20Agent%20tests.Tunneling%20MPLSoGRE.TCP%20-%20HTTP%20connection%20between%20containers%20on%20separate%20compute%20nodes%20succeeds.txt

http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/2737/TestReports/WindowsCompute/detailed_logs/Tunneling%20with%20Agent%20tests.Tunneling%20MPLSoGRE.UDP%20-%20sending%20message%20between%20containers%20on%20separate%20compute%20nodes%20succeeds.txt

These are logs for separate test cases, but the contents are redundant. The whole point of having these weird loggers was to separate them and I think it's a useful feature. I think we should strive to keep it this way